apps/aega/src/aega_meta_tx.erl

```diff
@@ -210,6 +212,23 @@ set_meta_result(Err = {error, _}, Tx, Trees, Env) ->
     {ok, Trees1, _Env} = aeprimop:eval(SetInstructions, Trees, Env),
     Trees1.
 
+-spec call_id(tx(), aec_trees:trees()) -> aect_call:id().
```

Comment:
This could be used in #2357 perhaps?

Reply:
I was wondering who should rebase, but I agree with you ;-)

Reply:
You and Thomas can fight that out I think ;-) I'll rebase on the end result regardless...

Reply:
@ThomasArts ^^

Reply:
One alternative is that we merge both, and then I'll resolve it in my upcoming PR...

Reply:
Looking at the two functions again (with a fresh cup of coffee) I think they are short enough and different enough not to require the extra effort...